backend/src/services/members/resolvers.ts

```diff
@@ -0,0 +1,18 @@
+import { IDbMember, getMember } from './db';
+import { UserContext } from '../../context';
+
+
+export default {
+  Query: {
+    async me({}, {}, context: UserContext) {
+      if (!context.user) return undefined;
+      const me = await getMember(context.user.stil_id || '');
```

Comment:
Hur ska ett tomt stilid tolkas? Om vi inte vill matcha något är det onödigt att göra ett databasanrop.

Reply:
Hur keycloak hanterar det är lite komplicerat så detta är en sak som borde diskuteras in person, för tillfället räknas det som att man inte har några poster.

Reply:
Jag tänker mer på nivån hur det tolkas i graphql-queryns semantik. Även en medlem utan poster lär ju ha ett stil-id. Så queryn borde bara anropas utan ett stil-id om personen inte är medlem (eller inte inloggad), och då borde getMembers databasanrop returnera null, om namnet reflekterar funktionaliteten.

Reply:
Nu hänger jag med, tycker inte någon match ska göras och därav blir det rimligt att lägga till även tomt stil-id som ett undantag i if-satsen.

Reply:
Beroende på användning är det till och med rimligt att kasta error om stil-id saknas.